style.css

```diff
@@ -0,0 +1,197 @@
+
+html {
```

Comment:
there's a number of indentation issues are, make sure that your spacing is correct.